boards/arm/reel_board/doc/index.rst

```diff
@@ -414,6 +519,15 @@ Then build and flash the application in the usual way.
    :board: reel_board
    :goals: build flash
 
+.. note::
+   Please use reel_board_v2 to build a application for the board equipped with
+   the `GDEH0213B72 display <#display>`_.
```

Comment:
change to (remove the #):
```
the `GDEH0213B72 display <display>`_.
```
This will link to the "Display" heading earlier in the doc.